examples/generator/mock_language/MockLanguageLexer.py

```diff
@@ -1,36 +1,35 @@
-# Generated from .\MockLanguage.g4 by ANTLR 4.9.1
+# Generated from MockLanguage.g4 by ANTLR 4.10.1
 from antlr4 import *
 from io import StringIO
-from typing.io import TextIO
 import sys
-
+if sys.version_info[1] > 5:
```

Comment:
Can you please add a comment on what/why this is doing? Other than that, everything looks good. Thank you!

Reply:
These files are automatically generated from antlr. This check was already present in 4.9.1 version in Parser and introduced since 4.9.2 in Lexer as well https://github.com/antlr/antlr4/pull/3067. The reason was that apparently `typing.io` is deprecated since Python 3.6.

Reply:
Sorry, forgot that that whole file is automatically generated.